src/components/NavBar.js

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 class NavBar extends React.PureComponent {
   render() {
     return (
       <>
         <div className="navbar">
           <h1 className="heading">Math Magicians</h1>
+          <div className="links">
+            <Link to="/">Home</Link>
+            <Link to="/calculator">calculator</Link>
+            <Link to="/Quotes">Quotes</Link>
+          </div>
         </div>
```

Comment:
- Optional
I would strongly recommend using `NavLink` instead of `Link` component. You can read more `NavLink` [here](https://reactrouter.com/en/main/components/nav-link).